types/queries/update_word.sql

```diff
@@ -8,6 +8,12 @@ update word set
         case
             when $3::text[] != '{}' then $3[1]
             else commentary
+        end,
+
+    english_translation =
+        case
+            when $2::text[] != '{}' and $2[1] is not null then $2[1]
```

Comment:
Each number in this file corresponds to an input parameter. Notice that in `types/src/database_sql.rs:515:27` you call this query with 4 parameters, but only three are defined here:
```
let document_id = query_file!(
    "queries/update_word.sql",
    word.id,            /// $1
    &source as _,       /// $2
    &commentary as _,   /// $3
    &translation as _,  /// $4? Does not exist!
)
```
Try replacing the reference to `$2` with `$4` instead!